web.browser/src/containers/PostList/PostListContainer.js

```diff
@@ -0,0 +1,61 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { updateVote, sortPopular, sortNewest } from '../../redux/actions';
+import { data } from '../../mock-data';
+
+import PostList from './PostList';
+import PostToolBar from '../../components/PostToolBar';
+
+class PostListContainer extends Component {
+  constructor() {
+    super();
+    this.updateVote = this.updateVote.bind(this);
+    this.sortPopular = this.sortPopular.bind(this);
+    this.sortNewest = this.sortNewest.bind(this);
+    this.state = {
+      posts: data.posts,
```

Comment:
I know we refactored this together, but please ensure you remove all local state from your components, when using Redux.